crates/vm/src/system/connector/mod.rs

```diff
@@ -145,6 +145,8 @@ impl<AB: InteractionBuilder + PairBuilder + AirBuilderWithPublicValues> Air<AB>
             .when_transition()
             .assert_eq(end.is_terminate, is_terminate);
 
+        builder.assert_zero(begin.timestamp);
```

Comment:
We could of course instead just send `0` in the interaction below, but I think this is clearer. 
Otherwise `begin.timestamp` would be unconstrained and potentially confusing.